src/models/recipe.model.ts

```diff
@@ -1,5 +1,88 @@
 import { Document, Schema, model } from "mongoose";
 
+/**
+ * @swagger
+ * components:
+ *   schemas:
+ *     Recipe:
+ *       type: object
+ *       properties:
+ *         title:
+ *            type: string
+ *            description: Recipe title. Min 3, max 80 chars
+ *            example: Chocolate cookies
+ *         description:
+ *            type: string
+ *            description: Recipe description. Not required. Max 256 chars.
+ *            example: My first recipe
+ *         preparing:
+ *            type: array of strings
+ *            description: Recipe preparing guide. Each step is new index in array.
+ *            example: ['Prepare ingredients', 'Measure flour', 'Set up the oven']
+ *         ingredients:
+ *            type: array of arrays of strings
+ *            description: Recipe ingredients. Proposed format is [['amount measure', 'ingredient'],['amount measure', 'ingredient']] but it is allowed to have only one item in nested array in case where amount and measure are not needed.
+ *            examples:
+ *              proposed: [['1tbs', 'salt'], ['2 cups', 'water'], ['3 cups', 'flour']]
+ *              optional: [['1 cup', 'water'], ['1tbs', 'basil'],['pepper'], ['salt']]
+ *         url:
+ *            type: string
+ *            description: Work in progress. Will be completely changed
+ *            example: /must/start/with/index.js
+ *       required:
+ *         - title
+ *         - preparing
+ *         - ingredients
+ *         - url
+ *
+ */
+
+/**
+ * @swagger
+ * components:
+ *   schemas:
+ *     RecipeDocument:
+ *       type: object
+ *       properties:
+ *         title:
+ *            type: string
+ *            description: Recipe title. Min 3, max 80 chars
+ *            example: Chocolate cookies
+ *         description:
+ *            type: string
+ *            description: Recipe description. Not required. Max 256 chars.
+ *            example: My first recipe
+ *         preparing:
+ *            type: array of strings
+ *            description: Recipe preparing guide. Each step is new index in array.
+ *            example: ['Prepare ingredients', 'Measure flour', 'Set up the oven']
+ *         ingredients:
+ *            type: array of arrays of strings
+ *            description: Recipe ingredients. Proposed format is [['amount measure', 'ingredient'],['amount measure', 'ingredient']] but it is allowed to have only one item in nested array in case where amount and measure are not needed.
+ *            examples:
+ *              proposed: [['1tbs', 'salt'], ['2 cups', 'water'], ['3 cups', 'flour']]
+ *              optional: [['1 cup', 'water'], ['1tbs', 'basil'],['pepper'], ['salt']]
+ *         url:
+ *            type: string
+ *            description: Work in progress. Will be completely changed
```

Comment:
As discussed - describe present behavior in the documentation, not how it will work in the future. Eventually you can inform that this field is `deprecated` but I think that's not the case here